_data/sidebars/marketplace_dev_sidebar.yml

```diff
@@ -1241,13 +1240,14 @@ entries:
                  - "202204.0"
 
           - title: "UI components library"
+            url: /docs/marketplace/dev/front-end/ui-components-library/
```

Comment:
This doesn't look right

Reply:
because the landing page is `index.md`, we can omit the file name here. The link works: https://docs.spryker.com/docs/marketplace/dev/front-end/202212.0/ui-components-library/